### DISCUSSION TITLE
Security headers for S3 based static website

### DISCUSSION THREAD

**Discussion by iamjatindersingh** [thuliteio/doks#404] [Q&A]
I am new to GoHUGO and testing deploying the DOKS on S3. In the code, I noticed there is an "index.headers" file with security headers. On build, this gets translated to "_headers". Will this file work for S3 based static website?

**Comment by h-enk** [comment]
I guess that won't work. Maybe this will help: https://medium.com/@netscylla/adding-security-headers-to-s3-websites-2002f243aa8f

**Reply by iamjatindersingh** [reply]
Thanks for the prompt reply! After learning more about the gohugo security headers, I ended up implementing a Lambda function. Works like a charm. Thanks.
